sources/Outputs/src/Dyd.cpp

```diff
@@ -185,18 +179,12 @@ Dyd::writeLoad(const algo::LoadDefinition& load, const std::string& basename) {
 boost::shared_ptr<dynamicdata::BlackBoxModel>
 Dyd::writeGenerator(const algo::GeneratorDefinition& def, const std::string& basename) {
   auto model = dynamicdata::BlackBoxModelFactory::newModel(def.id);
-  const std::string& static_ref_macro = (def.model == algo::GeneratorDefinition::ModelType::WITH_IMPEDANCE_SIGNALN ||
-                                         def.model == algo::GeneratorDefinition::ModelType::WITH_IMPEDANCE_DIAGRAM_PQ_SIGNALN)
-                                            ? macroStaticRefImpGeneratorName_
-                                            : macroStaticRefSignalNGeneratorName_;
+  const std::string& static_ref_macro = macroStaticRefSignalNGeneratorName_;
```

Comment:
If static_ref_macro always equal macroStaticRefSignalNGeneratorName_, use macroStaticRefSignalNGeneratorName_ directly without local variable

Reply:
Of course, thanks!